### ISSUE TITLE
synk-monitor version in Charts.yaml does not update with version in upstream repo

### ISSUE THREAD

**Issue by Joeskyyy** [snyk/kubernetes-monitor#387]
When new chart versions are published, they do not match the version in the upstream repo.

e.g. right now the Chart.yaml reflects version `0.1.0` but the upstream is `1.22.01

Small nit on my part here, but sometimes I like to go spelunking into the code (looking at `values.yaml` etc) and look at the chart version to know what to start playing with. As it stands right now, to understand what a specific chart version would have included in it, you would need to `helm fetch` the repo locally to do this inspection.

**Comment by ivanstanev** [comment]
Hey @Joeskyyy, I think the release process is not fully documented unless you go looking at the CircleCI config!
Right now we publish our Helm releases to GH Pages: https://snyk.github.io/kubernetes-monitor/snyk-monitor/
You can see all the releases of the chart by checking out the `gh-pages` branch: https://github.com/snyk/kubernetes-monitor/tree/gh-pages
All the released Charts will be listed there.

Basically once we release a Chart, we do not commit back to staging an updated version, the release process handles the right versioning when it's time for us to publish a new version of the chart or the product!

**Comment by shaimendel** [comment]
@Joeskyyy thank you for contributing and validating our product :)
please let us know if you have any further question, or we can close the issue otherwise

**Comment by Joeskyyy** [comment]
Hey folks,

I think we're mostly safe to close this out yeah. But I would maybe request updating the existing documentation to point to stuff like this in the README.md? It seems like a bit of the information is unintentionally hard to find and known only to the Snyk folks, so folks who want to contribute are a little limited in our understanding. 

Another example is the PR template links to dead links for the testing needed. I'm assuming those might just be internal only permission wise to folks on the Snyk team?

Regardless, happy to be *that* guy ;) Stay safe and healthy friends!
